### ISSUE TITLE
Bug: MAX

### ISSUE THREAD

**Issue by AdamK-DE** [foxthefox/ioBroker.fritzdect#486]
Hi,

per Fritz "SmartHome" APP Auf Android Eingestellter Werte werden bei MAX nicht übernommen.
MAX kommt in der App nach 28°C und nur bis 28°C nimmt der Adapter das auch und zeigt es in Objekten an.
Stellt man MAX ein dann springt er sogar runter - hier eben nahm er dann für "tsoll" das "lasttarget" und verstellt damit auch das Thermostat also stellt es wieder runter....

**Comment by foxthefox** [comment]
Ich bräuchte zum Nachvollziehen des Problems noch Infos.

- welche fritzdect Adapterversion?
- genauer Ablauf der Aktionen mit einer Angabe was passiert in iob oder der App und was ist erwartungskonform und was nicht
- für diesen Ablauf mal den Adapter in debug-Modus versetzen und das log schicken.

Kleine Hintergrundinfo, die FB verschlüsselt den Saunamodus (MAX) als Wert 254 in die gleiche Variable wie den Sollwert. Der Adapapter trennt dies auf in Temperaturen als Zahl und Zustände. Wenn also MAX oder MIN in der FB/APP eingestellt wird, bleibt es bei einer Temperatur im tsoll, diese wird mit default angesetzt. Denn MAX oder MIN sind nicht als Temperatur definiert. Der geänderte tsoll wird mit ACK=true in iob geschrieben und löst keine Aktion in Richtung FB aus.

**Comment by AdamK-DE** [comment]
**ahh! ne dann vergiss es wieder ;) kein bug!** 

ich ging davon aus das tsoll dann auch gesendet wird wenn es der adapter aktuallisiert.. konnte das mit der fritz app aber auch nicht nachvollziehen... dort bleibt es anscheinend bei MAX... per http://fritz.box kann man das dummerweise gar nicht sehen oder einstellen. seltsam..

passte meine VIS an. werde MAX dann gar nicht mehr nutzen sondern 28°C rüber jagen :) das passt und das wird dann auch in der VIS korrekt angezeigt. ansonsten hat man so wohl keine möglichkeit das "MAX" auch irgendwie zu erkennen?


PS: evtl als Anregung: echter "Thermostat hat x °C gemessen" DP wäre nicht verkehrt. in Celsius steht nur der angepasste mit dem offset. ich rechne das für die echte gemessene temp (oben neben der raum temp) nun um.. vorallem beim anpassen des offsets wäre das von vorteil... :

![Unbenannt](https://github.com/foxthefox/ioBroker.fritzdect/assets/127183545/a097ce90-a3da-4f00-b2ae-5b151eda569c)

**Comment by AdamK-DE** [comment]
ja, aber dann geht nur entweder oder.. also steuerung komplett per ioB ODER fritz.... 

ich schaue mir das die tage hier noch an und dann bastle ich evtl was für ioB zurecht.. heatingcontrol-adapter per blockly ;) ... bierher wuschelt ioB hier nur bei den fritz programmen mit rein und verändert tsoll bei night falls anwesend etc.. aber die zeitsteuerung übernimmt fritz...  bzw die thermostate ja selbst...

ohne offset ist dann auch sehr umständlich bzgl des einstellens der temp. naja, neue datenpunkte dann fällig .. das blockly wäre dann eh extrem aufwändig da es ja erstmal lernen müsste wie es tsoll ansteuern muss.. vorhin schaffte ichs selbst bei 28°C nicht über 21°C echte raum temp... ich spiele hier parallen auch an der vorlauftemp der gas.therme... sparen sparen sparen ;) .. wird dadurch alles etwas tricky...
